README.rst

```diff
@@ -58,6 +58,14 @@ corresponds to Pipefile.lock used (digest computed on ``Pipfile`` content).
 ``micropipenv`` requires toml extras for this functionality, so you will need
 to install ``micropipenv[toml]`` (see installation instructions bellow).
 
+You can supply additional positional arguments that will be passed to ``pip``.
+Use double dashes to distinguish ``pip`` options from ``micropipenv`` options.
+
+.. code-block::
+
+  # issue `pip install --user'
+  micropipenv install -- --user
```

Comment:
`  micropipenv install -- --user`, is -- here are supposed to be in this order?

Reply:
Yes, `--user` is an argument of micropipenv that is passed as an option to pip.